integration-tests/deployment/ccip/active_candidate_test.go

```diff
@@ -0,0 +1,243 @@
+	require.NoError(t, err)
+	tx, err = state.Chains[homeCS].CCIPHome.TransferOwnership(e.Env.Chains[homeCS].DeployerKey, state.Chains[homeCS].Timelock.Address())
+	require.NoError(t, err)
+	_, err = deployment.ConfirmIfNoError(e.Env.Chains[homeCS], tx, err)
+	require.NoError(t, err)
+
+	acceptOwnershipProposal, err := GenerateAcceptOwnershipProposal(state, homeCS, initialDeploy)
+	require.NoError(t, err)
+	acceptOwnershipExec := SignProposal(t, e.Env, acceptOwnershipProposal)
+	// Apply the accept ownership proposal to all the chains.
+	for _, sel := range initialDeploy {
+		ExecuteProposal(t, e.Env, acceptOwnershipExec, state, sel)
+	}
+	for _, chain := range initialDeploy {
+		owner, err2 := state.Chains[chain].OnRamp.Owner(nil)
+		require.NoError(t, err2)
+		require.Equal(t, state.Chains[chain].Timelock.Address(), owner)
+	}
+	cfgOwner, err := state.Chains[homeCS].CCIPHome.Owner(nil)
+	require.NoError(t, err)
+	crOwner, err := state.Chains[homeCS].CapabilityRegistry.Owner(nil)
+	require.NoError(t, err)
+	require.Equal(t, state.Chains[homeCS].Timelock.Address(), cfgOwner)
+	require.Equal(t, state.Chains[homeCS].Timelock.Address(), crOwner)
+
+	// check setup was successful
+	donID, err := DonIDForChain(state.Chains[homeCS].CapabilityRegistry, state.Chains[homeCS].CCIPHome, destCS)
+	require.NoError(t, err)
+	donInfo, err := state.Chains[homeCS].CapabilityRegistry.GetDON(nil, donID)
+	require.NoError(t, err)
+	require.Equal(t, 5, len(donInfo.NodeP2PIds))
+	require.Equal(t, uint32(4), donInfo.ConfigCount)
+	// [SETUP] done
+
+	// [ACTIVE ONLY, NO CANDIDATE] send successful request on active
+	seqNum := SendRequest(t, e.Env, state, homeCS, destCS, false)
+	require.Equal(t, uint64(1), seqNum)
+	// uncomment when offchain is fixed
+	//require.NoError(t, ConfirmExecWithSeqNr(t, e.Env.Chains[homeCS], e.Env.Chains[destCS], state.Chains[destCS].OffRamp, &startBlock, seqNum))
+	// [ACTIVE ONLY, NO CANDIDATE] done
+
+	// [ACTIVE, CANDIDATE] setup by setting candidate through cap reg
+	state, err = LoadOnchainState(e.Env, e.Ab)
+	require.NoError(t, err)
+
+	// delete the last node from the list of nodes.
+	// bootstrap node is first and will be ignored so we delete from the end
+	e.Env.NodeIDs = e.Env.NodeIDs[:len(e.Env.NodeIDs)-1]
+	nodes, err := deployment.NodeInfo(e.Env.NodeIDs, e.Env.Offchain)
+	require.NoError(t, err)
+
+	// this will construct ocr3 configurations for the
+	// commit and exec plugin we will be using
+	ocr3ConfigMap, err := BuildOCR3ConfigForCCIPHome(
+		e.Env.Logger,
+		state.Chains[destCS].OffRamp,
+		e.Env.Chains[destCS],
+		e.FeedChainSel,
+		tokenConfig.GetTokenInfo(e.Env.Logger, state.Chains[destCS].LinkToken),
+		nodes.NonBootstraps(),
+		common.BytesToAddress([]byte(rmnHomeAddress)),
+	)
+
+	require.NoError(t, err)
+
+	var mcmsOps []mcms.Operation
+	// this is titled "ExecPlugin", but it will work for any plugin you pass it
+	setCandidateMCMSOps, err := SetCandidateExecPluginOps(
```

Comment:
Can we rename the `ExecPlugin` then? `SetCandidatePluginOps` should be okay in this case.

Reply:
yea, those changes are in a different PR, I left the same comment [here](https://github.com/smartcontractkit/chainlink/pull/14815#discussion_r1804014752). If it's not updated in that PR, I can bring the change here after